website/blog/2023-04-24-backup-frequency.md

```diff
@@ -0,0 +1,127 @@
+  backed up data with junk almost right away.
+- If you want item versioning and extensive retention policies, the cost overheads for super-frequent backups can be prohibitive.
+
+While backup frequency will vary with each business, it’s generally not the case
+that a backup interval of “nearly 0ms” will make sense.
+
+## Technical Considerations: Microsoft Graph State Tokens
+
+One of the reasons to back up fairly frequently is the use of Microsoft Graph
+state tokens to show what has changed about your data. For example Corso only
+captures incremental changes during backups, only needing to store the items
+that have been updated or added since the last backup. It does this using
+[state tokens](https://learn.microsoft.com/en-us/graph/delta-query-overview#state-tokens)
+that it stores within the Microsoft 365 infrastructure to checkpoint the end of
+a backup. This token is used by the next backup invocation to see what has
+changed, including deletions, within your data.
+
+The exact expiry of state tokens isn’t published by Microsoft, but our
+observations show that if you are only backing up every few days, these tokens
+can expire. This will force a new full backup each time which is both
+unnecessary and costly (in terms of time and bandwidth but not
+storage because of Corso’s deduplication).
+
+You can therefore reduce data transmission overhead, improve backup performance, and reduce RPO, by backing up more frequently.
+
+## Cost Considerations: Storage Costs
+
+With the threat of ransomware and other malicious data corruption, it’s a great
+idea to store full backups with some frequency. This means that, if you want to
+have frequent backups ****with**** retention of older versions, you’re going to
+need a lot of storage.  
+
+Tools like Corso that use S3 object storage will be cheaper than most others.
+Further Corso not only deduplicates, packs, and compresses data, but it also has
+a ton of smarts to only capture incremental changes between backups but always
+present them as a full backup (topic for another blog post!). This ensures that
+even if you have 1000s of backups per user or SharePoint site, you will always
+see fast restores, minimal storage overhead, and always-full backups.
+
+For other tools, you should evaluate if it uses storage efficiently:
+
+- Since there’s a per-object storage cost with most S3 tiers, backups should bundle small items together
+- Backups should include compression and de-duplication to be as small as possible
+
+Take a look at some of our recent writing on selecting the best S3 storage tier
+(spoiler warning it’s probably Glacier IR) for your S3 backups.
+
+### You still haven’t answered my question: How often should you back up?
+
+Independent of whether it's Microsoft 365 or other systems, at least once a
+day. Probably about once every 8 hours. It will ensure your backups are
+incremental and that you don’t lose too much work in the event of an incident.
```

Comment:
```suggestion
day. Probably about once every 8 hours. It will ensure your backups are
efficient due to incremental data capture and that you don’t lose too much work in the event of an incident.
```